src/cookbook/networking/web-sockets.md

```diff
@@ -135,9 +133,9 @@ class MyApp extends StatelessWidget {
 
 class MyHomePage extends StatefulWidget {
   const MyHomePage({
-    super.key,
```

Comment:
This looks like a regression, using super parameters is something we want to encourage.